libc/test/src/pthread/CMakeLists.txt

```diff
@@ -38,6 +40,8 @@ add_libc_unittest(
     libc.src.pthread.pthread_mutexattr_setpshared
     libc.src.pthread.pthread_mutexattr_setrobust
     libc.src.pthread.pthread_mutexattr_settype
+  LINK_LIBRARIES
+    -pthread
```

Comment:
Your report in #89261 only specifies issues for libc/test/src/pthread/pthread_condattr_test.cpp. Are the changes to pthread_mutexattr_test or pthread_attr_test necessary?

Reply:
No, they are not. Removed.